python/tk_multi_workfiles/util.py

```diff
@@ -261,3 +261,38 @@ def get_template_user_keys(template):
         if key.shotgun_entity_type == "HumanUser":
             user_keys.add(key.name)
     return user_keys
+
+
+def resolve_filters(filters):
+    """
+    
+    When passed a list of filters, it will resolve strings found in the filters using the context
+    example: '{context.user}' could get resolved to {'type': 'HumanUser', 'id': 86, 'name': 'Philip Scadding'} 
```

Comment:
We should take the time at some point to refactor this in shotgunutils @manneohrstrom . Shall I create a ticket?